be/src/exec/es/es_scroll_parser.cpp

```diff
@@ -416,6 +416,33 @@ Status ScrollParser::fill_tuple(const TupleDescriptor* tuple_desc, Tuple* tuple,
             break;
         }
 
+        case TYPE_DECIMALV2: {
```

Comment:
I think decimal type  for ES table is meaningfullless

Reply:
OK. Make sense for me. Btw, I think we should give users some tips in DOC or throw Exception here In case of unexpected errors when using decimal for es scan

Reply:
Mainly because there is no decimal type in ES，I'm just wondering what problem the decimal type solves when creating a ES table. Can you describe your needs？